src/Turn.js

```diff
@@ -0,0 +1,16 @@
+class Turn {
+  constructor(guess, card) {
+    this.guess = guess;
+    this.card = card;
+  }
+
+  returnGuess = () => this.guess; 
+
+  returnCard = () => this.card;
+
+  evaluateGuess = () => this.guess === this.card.correctAnswer ? true : false;
+
+  giveFeedback = () => this.evaluateGuess() ? 'correct!' : 'incorrect!';
+}
+
```

Comment:
I really like these arrow functions and ternary syntax! Nice implementation of these new techniques.